Use @timbl's vision description (#255)
solid/solidproject.org: pages/home.md

        <div class="columns">
          <div class="column is-hidden-touch is-offset-1 is-4">
            <figure class="image">
              <img src="{{site.baseurl}}/assets/img/single-sign-on.svg" alt="[]" />
            </figure>
          </div>
          <div class="column is-offset-1 is-5">
            <h2 class="title">The last login you'll ever need</h2>
            <p class="content is-large">

>               Solid provides for the first time a <b>single global logon system</b>, so that when you log into any web site, instead of having  to log in with the usual 'f' and 'g' blue buttons, and then be tracked by Facebook and Google, you can log in with any Solid provider you trust, and that won't track you.

Reviewer: Not sure if we want to directly confront f and g.

Reviewer 2: Modified, to make them just examples

      <div class="container">
        <div class="columns">
          <div class="column is-offset-1 is-5">
            <h2 class="title">Like peas in a Pod</h2>
            <p class="content is-large">

>               Solid provides separate places ("<a href="{{site.baseUrl}}/faqs#pod" title="Frequently Asked Questions - what is a Pod?">pods</a>") for you to store your data, a bit like Dropbox and GDrive and iCloud, but you have the <b>power to share</b> any file or folder or contact or event or whatever <b>with anyone who has a solid id</b>.  You store your data in any place you trust (including your own computer).

Reviewer: ```suggestion
              Solid provides separate places ("<a href="{{site.baseUrl}}/faqs#pod" title="Frequently Asked Questions - what is a Pod?">pods</a>") for you to store your data, a bit like Dropbox and Google Drive and iCloud, but you have the <strong>power to share</strong> any file or folder or contact or event or whatever <strong>with anyone who has a Solid ID</strong>.  You store your data in any place you trust (including your own computer).
```

          </div>
          <div class="column is-offset-1 is-5">
            <h2 class="title">
              Apps that give you the freedom to move
            </h2>
            <p class="content is-large">

>               When Solid apps start, they just store all the data they need to work in your pod.  They don't store the data themselves at some computer run by the person who owns the app.  So by default <b>you control all the data</b> in your solid world.  This is a massive improvement to the privacy world.

Reviewer: ```suggestion
              When Solid apps start, they just store all the data they need to work in your pod.  They don't store the data themselves at some computer run by the person who owns the app.  So by default <strong>you control all the data</strong> in your Solid world.  This is a massive improvement to your privacy.
```

            </p>
            <p class="content is-large">

>               Solid app builders don't just store the data in your pod any old way.  They use solid standard formats.  This means that you and the people you work with can actually use many different apps — at the same time even — to do stuff.

Reviewer: ```suggestion
              Solid app builders don't just store the data in your pod any old way.  They use Solid standard formats.  This means that you and the people you work with can actually use many different apps – at the same time even – to do stuff.
```
